A Concrete Guide to React Native basics (#1372)
kamranahmedse/developer-roadmap: content/roadmaps/100-frontend/content/121-mobile-applications/100-react-native.md

<ResourceGroupTitle>Free Content</ResourceGroupTitle>
<BadgeLink colorScheme='blue' badgeText='Official Website' href='https://reactnative.dev/'>Official Website</BadgeLink>
<BadgeLink colorScheme='blue' badgeText='Official Docs' href='https://reactnative.dev/docs/getting-started'>Official Getting Started to React Native</BadgeLink>

> <BadgeLink badgeText='Watch' href='https://www.udemy.com/course/react-native-tutorial/'>Learn the basics of React Native in a nice and easy way.</BadgeLink>

Reviewer: ```suggestion
<BadgeLink badgeText='Watch' href='https://www.youtube.com/watch?v=0-S5a0eXPoc'>Build a React Native App by Mosh</BadgeLink>
```
